collector/collector.go

```diff
@@ -78,11 +80,22 @@ func (c *collector) Collect(from, to time.Time) error {
 }
 
 func (c *collector) CollectTodaysMetrics() error {
+	todaysLatencies, err := c.Source.TodaysLatencies()
+	if err != nil {
+		return err
```

Comment:
Perhaps an error in latency data retrieval should not cause the collection of all other metrics to be skipped.

Reply:
Updated to prevent failure or either counts or latency collection from stopping collection of other metrics